packages/admin/cms-admin/src/pages/createEditPageNode.tsx

```diff
@@ -150,6 +150,11 @@ export function createEditPageNode({
                     // The unchanged slug is expected to be available
                     return "Available";
                 }
+                if (parentNodeData?.pageTreeNode?.path != undefined && newSlug === "home") {
+                    // The slug "home" is reserved for the home page
+                    return "Reserved";
+                }
```

Comment:
This is covered by the query, so I wouldn't add it here.

Reply:
removed it